### PR TITLE
fix: Bug fix in style and verification

### PR DESCRIPTION
**Pull request by cizil-keyvalue**
1. Fix retry verification token
2. Fix custom styles
3. Fix read me 
4. Update snapshot

### CODE REVIEW

File: src/components/NotificationIcon.vue [hunk 1]
```diff
@@ -1,29 +1,19 @@
...
-              width: 0,
-              height: 0,
-              overflow: 'hidden',
-            }
-      "
-    >
+    <div v-if="!hideBadge" class="siren-sdk-notificationIcon-badge-container" :style="badgeType === BadgeType.DEFAULT && unViewedCount > 0
+      ? styles.badgeStyle
+      : {
+        width: 0,
```
Comment (bilta-keyvalue) [inline src/components/NotificationIcon.vue]: move this to style

File: src/components/NotificationIcon.vue [hunk 2]
```diff
@@ -1,29 +1,19 @@
...
+        height: 0,
+        overflow: 'hidden',
+      }
+    ">
       <div :style="styles.badgeTextStyle">
         {{ unViewedCount > 99 ? "99+" : unViewedCount }}
```
Comment (bilta-keyvalue) [inline src/components/NotificationIcon.vue]: move the value 99 to a constant and manipulate the string from the constant

File: src/components/NotificationIcon.vue [hunk 3]
```diff
@@ -62,6 +52,7 @@ const props = defineProps<{
   darkMode: boolean;
   styles: SirenStyleProps;
   hideBadge: boolean;
+  showNotifications: boolean;
```
Comment (bilta-keyvalue) [inline src/components/NotificationIcon.vue]: use a more generic name

File: src/components/NotificationIcon.vue [hunk 4]
```diff
@@ -62,6 +52,7 @@ const props = defineProps<{
   darkMode: boolean;
   styles: SirenStyleProps;
   hideBadge: boolean;
```
Comment (bilta-keyvalue) [inline src/components/NotificationIcon.vue:47]: can't we manipulate the hidebadge from showNotifications prop

File: src/components/SirenInbox.vue [hunk 5]
```diff
@@ -1,11 +1,14 @@
...
-        <NotificationIcon :handleNotification="handleNotification"
-          :badgeType="showNotifications ? BadgeType.NONE : BadgeType.DEFAULT"
-          :darkMode="darkMode" :styles="styles" :hideBadge="hideBadge" />
-      </div>
-      <div v-if="showNotifications || windowViewOnly" :style="{
+  <div :class="windowViewOnly ? 'siren-sdk-window-container' : 'siren-sdk-inbox-container'" ref="notificationRef"
+    data-testid="siren-inbox-container">
+    <div v-if="!windowViewOnly" ref="iconRef" data-testid="siren-inbox-icon">
+      <NotificationIcon :handleNotification="handleNotification"
+        :badgeType="showNotifications ? BadgeType.NONE : BadgeType.DEFAULT"
```
Comment (bilta-keyvalue) [inline src/components/SirenInbox.vue]: rhis prop can be manipulated from showNootifications

**Review by bilta-keyvalue — APPROVED** [review]

**Review by nikhin-keyvalue — APPROVED** [review]

**Pull request merged**